src/main/java/com/minecolonies/compatibility/tinkers/SlimeTreeProxy.java

```diff
@@ -0,0 +1,27 @@
+package com.minecolonies.compatibility.tinkers;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import org.jetbrains.annotations.NotNull;
+
+public class SlimeTreeProxy
+{
+
+    private static final double SLIME_BLOCK_SLIPPERINESS = 0.5D;
+
+    /**
+     * Check if tree is Slime Tree.
+     *
+     * @param world the world.
+     * @param pos the position
+     * @return if the tree is Slime Tree.
+     */
+    protected boolean checkForTinkersSlimeBlock(@NotNull final IBlockAccess world, @NotNull final BlockPos pos)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "world". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)